docs/FeatureManagement/FeatureFlagv2.0.0.schema.json

```diff
@@ -0,0 +1,354 @@
+            "items": {
+              "$id": "#/properties/conditions/properties/client_filters/items",
+              "type": "object",
+              "title": "Client Filter",
+              "required": [
+                "name"
+              ],
+              "properties": {
+                "name": {
+                  "$id": "#/properties/conditions/properties/client_filters/items/properties/name",
+                  "type": "string",
+                  "title": "Client Filter Name",
+                  "description": "The name used to refer to and require a client filter.",
+                  "default": "",
+                  "examples": [
+                    "Percentage",
+                    "TimeWindow"
+                  ],
+                  "pattern": "^(.*)$"
+                },
+                "parameters": {
+                  "$id": "#/properties/conditions/properties/client_filters/items/properties/parameters",
+                  "type": "object",
+                  "title": "Client Filter Parameters",
+                  "description": "Custom parameters for a given client filter. A client filter can require any set of parameters of any type.",
+                  "required": [],
+                  "patternProperties": {
+                    "^.*$": {
+                      "anyOf": [
+                        {
+                          "type": "string"
+                        },
+                        {
+                          "type": "null"
+                        },
+                        {
+                          "type": "object"
+                        },
+                        {
+                          "type": "number"
+                        },
+                        {
+                          "type": "array"
+                        },
+                        {
+                          "type": "boolean"
+                        }
+                      ]
+                    }
+                  },
+                  "additionalProperties": false
+                }
+              }
+            }
+          }
+        }
+      },
+      "variants": {
+        "$id": "#/properties/variants",
+        "type": "array",
+        "title": "Variants Collection",
+        "description": "The list of variants defined for this feature. A variant represents a value of a feature flag that can be a string, a number, a boolean, or even a configuration object.",
+        "items": {
+            "$id": "#/properties/variants/items",
+            "type": "object",
+            "title": "Variant",
+            "required": [
+                "name"
+            ],
+            "properties":{
+                "name": {
+                    "$id": "#/properties/variants/items/properties/name",
+                    "type": "string",
+                    "title": "Variant Name",
+                    "description": "The name used to refer to a feature variant.",
+                    "default": "",
```

Comment:
It shouldn't have default if it's required.

Reply:
Yea I see now that I mixed up the defaults on a few properties, I'll fix them.